examples/inferring-isolated/inferring_isolated.description

```diff
@@ -1,10 +1,10 @@
 // `isolated` is a complex feature, which would be a lot for an application developer to understand.
 // A typical Ballerina application consists of a single module that imports multiple library modules.
-// Within a single module, we can infer `isolated` qualifiers. Object w/o mutable fields is
-// inherently `isolated`. Application developer's responsibility is to use `lock` statement where
+// Within a single module, the compiler infers `isolated` qualifiers. An object without mutable fields is
+// inherently `isolated`. It is the application developer's responsibility to use `lock` statements where
 // needed. e.g.,
 // <ul>
-// <li>access self in a `service` object with mutable state</li>
+// <li>access `self` in a `service` object with mutable state</li>
 // <li>access mutable module-level variables</li>
 // </ul>
```

Comment:
Also, another small thing. Shouldn't these be "accessing" instead of "access"? 🤔 These are examples of when to use lock statements right? So when you are reading this, wouldn't it be more natural if the examples are phrased like "accessing mutable module-level variables"? @MaryamZi thoughts?

Reply:
+1 for "accessing".